PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -88,14 +89,22 @@ class SearchViewModel: ObservableObject {
             image: .warning,
             title: Localization.Search.limitedResults,
             detail: Localization.Search.Banner.errorMessage,
-            action: BannerModifier.BannerData.BannerAction(
+            action: reportButton(with: featureFlags)
+        )
+        return isOffline ? offlineView : errorView
+    }
+
+    /// Handles whether to show a report button for a banner
+    private func reportButton(with featureFlagsService: FeatureFlagServiceProtocol) -> BannerAction? {
```

Comment:
The alternative was to make it so that the `reportButton` function is decoupled from the feature flags implementation. But now that I am looking at it, it might be better to remove it from the signature. @CMasterson we discussed this, any opposition to remove the feature flag service from the method signature since we discussed this previously?